litmus-portal/frontend/src/pages/EditSchedule/Schedule.tsx

```diff
@@ -0,0 +1,778 @@
+                            </div>
+                          ) : (
+                            <></>
+                          )}
+                          <FormControlLabel
+                            value="everyWeek"
+                            control={
+                              <Radio
+                                classes={{
+                                  root: classes.radio,
+                                  checked: classes.checked,
+                                }}
+                              />
+                            }
+                            label={t(
+                              'createWorkflow.scheduleWorkflow.every.week'
+                            )}
+                          />
+                          {valueDef === 'everyWeek' ? (
+                            <div>
+                              <div className={classes.scRandom}>
+                                <Typography className={classes.scRandsub1}>
+                                  {t('createWorkflow.scheduleWorkflow.on')}
+                                </Typography>
+                                <FormControl className={classes.formControlDT}>
+                                  <Select
+                                    className={classes.select}
+                                    disableUnderline
+                                    value={days}
+                                    onChange={(e) => {
+                                      setCronValue({
+                                        ...cronValue,
+                                        month: '*',
+                                        day_week: ((e.target
+                                          .value as unknown) as string).slice(
+                                          0,
+                                          3
+                                        ),
+                                      });
+                                      setDays(
+                                        (e.target.value as unknown) as string
+                                      );
+                                      workflowAction.setWorkflowDetails({
+                                        scheduleInput: {
+                                          ...workflowData.scheduleInput,
+                                          weekday: (e.target
+                                            .value as unknown) as string,
+                                        },
+                                      });
+                                    }}
+                                    label="days"
+                                    inputProps={{
+                                      name: 'days',
+                                      style: {
+                                        fontSize: '0.75rem',
+                                        height: 7,
+                                      },
+                                    }}
```

Comment:
I'm changing the unit less value to rem but I can't add `classes.<styles>` since it's an inputProp property inside Select component and input Props only takes an object as an argument. useStyle is of type makeStyle by MUI, it is type incompatible.